.travis.yml

```diff
@@ -80,6 +80,8 @@ jobs:
           && npm init -y && npm install --save-dev @commitlint/travis-cli && popd
         # yamllint enable rule:line-length
         - export PATH=/opt/commitlint/node_modules/.bin:$PATH
+        # install pylint for python files
+        - apt-get install python3-pylint
```

Comment:
probably need to add python3-prettytable here as well

Reply:
It also seems this needs `sudo`...

Reply:
Yes, there was a permission issue; apt doesn't directly support non-root user.